near/res/testnet_genesis_config.json

```diff
@@ -1,6 +1,7 @@
 {
   "protocol_version": 4,
   "config_version": 1,
```

Comment:
Change config_version

Reply:
I would postpone doing this until we figure out the story with version updates.